### PR TITLE
add README

### PR DESCRIPTION
**Pull request by bam241**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+*******
+UnitBot
+*******
+
+
+`unitBot` is a slack bot that convert imperial unit into european unit, and vice
```
Comment (pshriwise) [inline README.md]: convert to converts

File: README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+versa.
+
+To run one need pivate.py file which contains (see example below):
```
Comment (pshriwise) [inline README.md]: need to needs

File: README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- api token,
+- a user name or channel name to send the error messages.
+
+``` API_TOKEN= 'my_slack_group_token' ERRORS_TO= 'where_to_send_erorr' ```
+
```
Comment (pshriwise) [inline README.md]: Maybe include here that users could also provide the API token in a `SLACKBOT_API_TOKEN` environment variable.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+unitBot depends on the `quantities` python library as well as slackbot.
+
+The `unibot_plugins.py` contains the parameters of the `unitbot`.
+
+To add new unit conversion, add lines in the ` unit_dict`, using the following
+syntax: `'X': [q.X, q.Y],`, where `X` reprensent the unit symbol, `q.X` the
```
Comment (pshriwise) [inline README.md]: maybe specify that 'X' represents the unit symbol the bot should recognize

File: README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+corresponding `quantities` unit and `q.Y` the `quantities` output unit.
+
+
+**Note**: if using non absolut value (such as `Celcius`), conversion might need some
```
Comment (pshriwise) [inline README.md]: absolute 

and conversion to the conversion

**Review by pshriwise — COMMENTED** [review]
A few comments on the english (really very good though). And a detail that might help others contribute.

**Comment by bam241** [comment]
@pshriwise done !

File: README.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
+*******
+UnitBot
+*******
+
+
+`unitBot` is a slack bot that converts imperial unit into european unit, and vice
```
Comment (pshriwise) [inline README.md]: unit to units (for both)

File: README.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+versa.
+
+To run one needs pivate.py file which contains (see example below):
```
Comment (pshriwise) [inline README.md]: needs a

File: README.md [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+- api token,
+- a user name or channel name to send the error messages.
+
+``` 
+API_TOKEN= 'my_slack_group_token' 
+ERRORS_TO= 'where_to_send_erorr' 
+```
+**Note**: The API token also provide the API token in a `SLACKBOT_API_TOKEN
```
Comment (pshriwise) [inline README.md]: The API token can also be provided...

Comment (pshriwise) [reply README.md]: Also second back tick might be in the wrong place?

File: README.md [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+environment variable`.
+
+unitBot depends on the `quantities` python library as well as slackbot.
+
+The `unibot_plugins.py` contains the parameters of the `unitbot`.
+
+To add new unit conversion, add lines in the ` unit_dict`, using the following
+syntax: `'X': [q.X, q.Y],`, where `X` represent the unit symbol that `uniBot`
```
Comment (pshriwise) [inline README.md]: represent to represents

File: README.md [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+should recognize, `q.X` the corresponding `quantities` unit and `q.Y` the
```
Comment (pshriwise) [inline README.md]: comma after unit here (personal preference)

File: README.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+`quantities` output unit.
+
+
+**Note**: if using non absolute value (such as `Celcius`), the conversion might need some
```
Comment (pshriwise) [inline README.md]: using a non absolute

**Review by pshriwise — COMMENTED** [review]
Couple more corrections I missed the first time.

File: README.md (hunk 10 above)
Comment (bam241) [reply README.md]: I usually don't do it because my wife bit me for this... I will add it, but if she ever ask this is you comma :)

**Comment by bam241** [comment]
I added the float management, 
also fix the english.

**Review by pshriwise — APPROVED** [review]
Looks great @bam241!

**Pull request merged**